velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -1383,6 +1385,85 @@ TpchPlan TpchQueryBuilder::getQ16Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ17Plan() const {
+  std::vector<std::string> lineitemColumns = {
+      "l_partkey", "l_extendedprice", "l_quantity"};
+  std::vector<std::string> partColumns = {
+      "p_partkey", "p_brand", "p_container"};
+
+  auto lineitemSelectedRowType = getRowType(kLineitem, lineitemColumns);
```

Comment:
nit: drop 'Selected' for brevity: lineitemRowType and partRowType

Reply:
Done, thanks!